mk/install.sh

```diff
@@ -228,6 +228,21 @@ select_pkgs_xe()
     select_rpm_utilities
 }
 
+select_pkgs_freebsd()
+{
+    GUEST_PKG_TYPE=pkg
+    PKGNAME=$(pkg info xe-guest-utilities 2>&1)
+    ERROR=$?
+    if [ $ERROR != 0 ]
+    then
+    	XGU="xe-guest-utilities"
```

Comment:
I appreciate the zeal, but that isn't the case. This file is chock full of tab/space combinations. Although most lines (280) indent with only spaces, almost that many (193) indent with an initial tab and then use spaces 🙄. Some sections of code (lines 646-658) freely mix tabs and spaces (line 648 is a tab and 4 spaces; the next line is 2 tabs). I will commit a new version where *my* code only has spaces, but it doesn't fix the file.

Reply:
Well at least let's stay consistent with what is around the portions of codes that we modify. I'd fix the whole file if there was a way to contribute the fix to Citrix, but there isn't for this specific file so that would make patching a nightmare whenever a new one is released by them :)